integrations/terraform/Makefile

```
@@ -11,8 +11,21 @@ BUILDFLAGS ?= $(ADDFLAGS) -trimpath -ldflags '-w -s'
 # CGO must NOT be enabled as hashicorp cloud does not support running providers using on CGO.
 CGOFLAG ?= CGO_ENABLED=0
 
+# base name for release archive (used for .tar.gz packaging across all platforms)
 RELEASE = terraform-provider-teleport-v$(VERSION)-$(OS)-$(ARCH)-bin
 
+# executable extension (.exe) for windows specific builds
+EXEEXT =
+ifeq ($(OS),windows)
+  EXEEXT = .exe
+endif
+
+# filename of the compiled binary $(BUILDDIR)
+BINARY_NAME = terraform-provider-teleport$(EXEEXT)
+
+# windows terraform provider filename as it will appear on downloads page
+WINDOWS_DOWNLOAD_FILENAME := terraform-provider-teleport-v$(VERSION)-$(OS)-$(ARCH)$(EXEEXT)
```

The OS should always be "windows" here, right?
```suggestion
WINDOWS_DOWNLOAD_FILENAME := terraform-provider-teleport-v$(VERSION)-windows-$(ARCH)$(EXEEXT)
```